### DISCUSSION TITLE
Kepler for OTEL receiver

### DISCUSSION THREAD

**Discussion by alan-cueva** [sustainable-computing-io/kepler#1092] [Q&A] [answered]
Hello all,

I am new in Kepler and I want to create an OTEL collector that use Kepler in the receiver (then send it to whatever I need), the problem is that I am not sure if it is possible, I am seeing that Kepler exports metrics to Prometheus. Is it possible export in OTEL format?

**Comment by rootfs** [comment]
@husky-parul @sthaha

**Comment by sthaha** [comment, marked as answer]
@alan-cueva , @husky-parul and I are working on switching to otel-sdk so that kelper can export in otel - http/grpc format or as prometheus metrics. 
Meanwhile, have you tried using a prometheus receiver? 
An example can be seen here - https://github.com/danielm0hr/edge-metrics-measurements/blob/main/scenario_03/compose.yml

**Reply by alan-cueva** [reply]
Unfortunately, I need something very specific, I will be waiting for the OTEL version. Thanks so much.

**Reply by husky-parul** [reply]
@alan-cueva Could you expand on the requirements? I understand you want to consume Kepler metrics in OTLP format. Could you describe your collector pipeline?

**Reply by alan-cueva** [reply]
sorry for the very late reply. My project was cancelled, so we won't continue with this. The last think that we were working was exporting  data to prometheus and use its receiver to send it to Kafka, but as I mentioned it does not matter since everything was cancelled. Thanks anyway.
